cg/meta/workflow/mutant/mutant.py

```diff
@@ -188,12 +202,19 @@ def run_analysis(self, case_id: str, dry_run: bool, config_artic: str = None) ->
             )
 
     def get_cases_to_store(self) -> list[Case]:
-        """Return cases where analysis has a deliverables file,
-        and is ready to be stored in Housekeeper."""
+        """Return cases for which the analysis is complete on Traiblazer and a QC report has been generated."""
+        return [
+            case
+            for case in self.get_cases_with_completed_analysis_and_not_stored()
+            if self.get_case_qc_report_path(case_id=case.internal_id).exists()
+        ]
+
+    def get_cases_with_completed_analysis_and_not_stored(self) -> list[Case]:
```

Comment:
Looking for tips on the naming of this function!

Reply:
How about:
```
get_cases_with_completed_not_stored_analysis

Reply:
Thanks! Will change to that!